index.html

```diff
@@ -24,10 +24,9 @@
     </header>
     <main>
       <!--1st section - Hero -->
-      <section class="section">
+      <section class="hero">
```

Comment:
You can leave the common class "section" in the section tag, and apart from that add the second class name "hero" and put in "hero" only what varies from the normal "section" styling.